x-pack/legacy/plugins/rollup/server/lib/search_strategies/rollup_search_capabilities.js

```diff
@@ -22,12 +22,7 @@ export const getRollupSearchCapabilities = (DefaultSearchCapabilities) =>
     }
 
     get defaultTimeInterval() {
-      return this.dateHistogram.fixed_interval || this.dateHistogram.calendar_interval ||
-        /*
-          Deprecation: [interval] on [date_histogram] is deprecated, use [fixed_interval] or [calendar_interval] in the future.
-          We can remove the following line only for versions > 8.x
-         */
-        this.dateHistogram.interval || null;
```

Comment:
Will you make sure when backporting this, that the `this.dateHistogram.interval` will be added to the backport? Because I think we should prior to 8.0 still check for the interval itself.

Reply:
@timroes  I think it's a wrong comment. If I understand propertly 'interval' was removed from ES v.7.3.  Am I right? If yes, not sure that we need that check now.

Reply:
@timroes Thanks! Fixed